user_vocab.py

```diff
@@ -10,12 +10,12 @@
     with open(filename) as f:
         content = f.read().splitlines()
 
-def rmv_non_charac(word):
+def remove_non_character(word):
     edited_word = [character for character in word if ord(character) in range (97, 123)]
     if len("".join(edited_word)) > 0:
         return "".join(edited_word) 
 
 unclean_vocab = sorted({words.lower() for words in content if " " not in words}) #issue here: as single words with spaces excluded
-clean_vocab = {rmv_non_charac(word) for word in unclean_vocab}
+clean_vocab = {remove_non_character(word) for word in unclean_vocab}
```

Comment:
Further to this; what you are trying to do is known as santizing a string.

A function name of "sanitize" is broad,  but acceptable. Put an internal docstring explaining the sanitisation rules.